spring-cloud-task-stream/src/main/java/org/springframework/cloud/task/batch/listener/support/ExitStatus.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright 2016 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.task.batch.listener.support;
+
+/**
+ * ExitStatus DTO created so that {@link org.springframework.batch.core.ExitStatus} can be serialized into Json without
+ * having to add mixins to an ObjectMapper
+ * @author Glenn Renfro
+ */
+public class ExitStatus {
+
+	private String exitCode;
+
+	private String exitDescription;
+
+	public ExitStatus(){
+
+	}
+
+	public ExitStatus(String exitCode, String exitDescription) {
```

Comment:
Why do we have this constructor?  I'd expect a `public ExitStatus(ExitStatus status)`...